### PR TITLE
docs: minor changes to docs and tutorials, rearrange some things

### PR DESCRIPTION
**Pull request by HarikrishnanBalagopal**
Signed-off-by: Harikrishnan Balagopal <harikrishmenon@gmail.com>

### CODE REVIEW

File: docs/tutorials/customizing-the-output/ingress-annotator.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
 ---
 layout: default
-title: "Ingress annotator"
-permalink: /tutorials/migration-workflow/ingress-annotator
-parent: "Migration workflow"
+title: "Adding annotations to Ingress"
```
Comment (ashokponkumar) [inline docs/tutorials/customizing-the-output/ingress-annotator.md:3]: Do we want to call the task as adding annotations to ingress or 'adding custom annotations to kubernetes yamls' and in the contents we can state it as for example ingress

File: docs/tutorials/customizing-the-output/script-inject.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
 ---
 layout: default
-title: "Custom transformer for injecting a script into the container image"
-permalink: /tutorials/migration-workflow/script-inject
-parent: "Migration workflow"
+title: "Injecting a script into the container image"
```
Comment (ashokponkumar) [inline docs/tutorials/customizing-the-output/script-inject.md:3]: Do we want to call it as something like, adding custom files to source

**Review by ashokponkumar — CHANGES_REQUESTED** [review]
Small suggestions

File: docs/tutorials/customizing-the-output/ingress-annotator.md (hunk 1 above)
Comment (HarikrishnanBalagopal) [reply docs/tutorials/customizing-the-output/ingress-annotator.md]: done

File: docs/tutorials/customizing-the-output/script-inject.md (hunk 2 above)
Comment (HarikrishnanBalagopal) [reply docs/tutorials/customizing-the-output/script-inject.md]: done

**Pull request merged**